MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1753,6 +1753,21 @@ public bool CanStreamCopyVideo(EncodingJobInfo state, MediaStream videoStream)
                 }
             }
 
+            var requestedRangeTypes = state.GetRequestedRangeTypes(videoStream.Codec);
+            if (requestedProfiles.Length > 0)
+            {
+                if (string.IsNullOrEmpty(videoStream.VideoRangeType))
+                {
+                    return false;
+                }
+
+                if (!string.IsNullOrEmpty(videoStream.VideoRangeType)
+                    && !requestedRangeTypes.Contains(videoStream.VideoRangeType, StringComparison.OrdinalIgnoreCase))
```

Comment:
```suggestion
                if (!requestedRangeTypes.Contains(videoStream.VideoRangeType, StringComparison.OrdinalIgnoreCase))
```

`videoStream.VideoRangeType` will always have a value here